src/app/template/shared/template.service.ts

```diff
@@ -20,11 +21,12 @@ export class TemplateService extends BaseTemplateService {
 
   public register(params: RegisterTemplateBaseParams): Observable<Template> {
     // stub
-    params['hypervisor'] = 'KVM';
-    params['format'] = 'QCOW2';
-    params['requiresHvm'] = true;
+    const requestParams = cloneDeep(params);
+    requestParams['hypervisor'] = 'KVM';
```

Comment:
Why don't we check if those params are missing?
I believe that we have an option to specify hypervisor, format, requiresHvm on the creation form